.travis.yml

```diff
@@ -22,3 +22,6 @@ install:
 
 script:
   - cd ${TRAVIS_BUILD_DIR} && make test
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
Do you need to ask travis to create the coverage profiles or does the bash script automatically generate them?

Reply:
This seems to be the canonical way to invoke codecov (and I cannot find a profile option).

Reply:
The current cov report is unknown after merging this PR. I guess we do need go test to create the coverage profiles. cc @sameo

Reply:
Looks like it might be a permissions issue?